source/meetings/2018/february/index.html.md

```diff
@@ -34,29 +34,24 @@ minutes.  This year our talks are:
 > harder to change and annoying to work with. There are human costs to these
 > decisions we make every day that have implications beyond the codebase.
 
+{::coverage year="2018" month="february" talk="the-human-cost-of-messy-code" /}
+
 ## Search Relevancy
 
 [Rosa Fox](https://twitter.com/rosaemerald) says:
 
 > A quick introduction to search result relevancy featuring GOV.UK.
 
+{::coverage year="2018" month="february" talk="search-relevancy" /}
+
 ## \_why Symbol#to\_proc works and more
 
 [Simon George](https://twitter.com/sfcgim) says:
 
 > How `Symbol#to_proc` found its way into Ruby core, who put it there, ways it
 > can be extended, and a crude drawing of a fish.
 
-## Software Development – a view from the Boardroom
```

Comment:
I can't remember this far back to know if this talk didn't get recorded or didn't happen, but we should probably leave it in unless we have other evidence that it didn't happen.  In the past we've put striketthrough styling on it to show that kind of thing rather than remove it entirely.  Can we revert this delete?

Reply:
Hey @h-lame! So the reason I removed this one from the February talk is that it was also in May of the same year. If you can remember which month it happened in I'll put it into the correct folder md file :).

Reply:
Just checked the May meeting and you're right, I guess Alan had to pull out for Feb and we rescheduled for May.  Looking at the video from May he spoke for ~30 minutes so definitely not a lightning talk which is all we'd have allowed in Feb.  You've done the right thing here to remove him from the Feb meeting page.  Thanks for being so diligent!  I imagine there might be a bunch more of this kind of thing, but probably hard to find them all.  I'll resolve this comment thread now.